src/app/services/tests/ConnectedUserService.spec.ts

```diff
@@ -369,9 +391,9 @@ describe('ConnectedUserService', () => {
             expect(service.createUser).toHaveBeenCalledWith(user.uid);
         });
         it('should not create the user if it already exists', async() => {
-            // given a google user
+            // given a disconnected google user
             spyOn(service, 'createUser');
-            const user: FireAuth.User = await createGoogleUser(true);
+            const user: FireAuth.User = await createDisconnectedGoogleUser('foo@bar.com');
```

Comment:
comment for the following line:
"then it *should* have succeed"

Eh, t'as un peu un baby test après le then haha :D

Reply:
changé ! baby test, what?